daemon/graphdriver/windows/windows.go

```diff
@@ -412,7 +412,24 @@ func (d *Driver) Put(id string) error {
 }
 
 // Cleanup ensures the information the driver stores is properly removed.
+// We use this opportunity to cleanup any -removing folders which may be
+// still left if the daemon was killed while it was removing a layer.
 func (d *Driver) Cleanup() error {
+	// Note we don't return an error on cleanup failure, but do log a warning.
+	items, err := ioutil.ReadDir(d.info.HomeDir)
+	if err != nil {
+		return nil
+	}
+	for _, item := range items {
+		if item.IsDir() && strings.HasSuffix(item.Name(), "-removing") {
+			if err := hcsshim.DestroyLayer(d.info, item.Name()); err != nil {
+				logrus.Warnf("Failed to cleanup %s: %s", item.Name(), err)
+			} else {
+				logrus.Infof("Cleaned up %s", item.Name())
```

Comment:
I think these logs are correct here. Will make sense with the updated comment. It's not debugging information.